pootle/apps/pootle_project/models.py

```diff
@@ -30,11 +30,12 @@
 from pootle.core.cache import make_method_key
 from pootle.core.mixins import CachedTreeItem
 from pootle.core.models import VirtualResource
-from pootle.core.url_helpers import (get_editor_filter, get_path_sortkey,
-                                     split_pootle_path, to_tp_relative_path)
```

Comment:
FWIW it has to be said there's no extra code being added: for small as it is, it's [reusing it](https://github.com/translate/pootle/blob/e62a1eff750d818f0c2071effe0cd2ed3462b79b/pootle/apps/pootle_app/models/directory.py#L93) instead of duplicating it. Again, I consider important to denote the _intentionality_, especially for anyone who is not familiar with the code, which is not quite clear by reading the `split` IMHO.